### PR TITLE
added text utilities and lists

### PR DESCRIPTION
**Pull request by sanjay-xdr**
added text utilities and lists
list-1
list-2
text-uti-1
text-util-2
text-util-3

### CODE REVIEW

File: Component/List/list.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  width: 30rem;
+  font-size: 4rem;
+  margin-left: 1rem;
+  padding: 1rem;
+  border-bottom: 1px solid var(--primary-color);
+}
+
+.list-item:hover {
+  background-color: rgba(205, 104, 245, 0.2);
+}
```
Comment (arjunbharti) [inline Component/List/list.css:22]: Use color variables

File: Component/Text Utilities/textUtilities.html [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </code>
+        </div>
+        <h3 class="font-medium margin-m2">Paragraphs</h3>
+
+        <div class="box">
```
Comment (arjunbharti) [inline Component/Text Utilities/textUtilities.html:156]: Remove extra spacing to make the code more cleaner.

File: Component/Text Utilities/textUtilities.html [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </code>
+        </div>
+
+        <h3 class="font-medium margin-m2">Center Text</h3>
+
```
Comment (arjunbharti) [inline Component/Text Utilities/textUtilities.html:185]: Same extra spacing issue there too.

File: Component/Text Utilities/textUtilities.html (hunk 2 above)
Comment (sanjay-xdr) [reply Component/Text Utilities/textUtilities.html:156]: yeah some time prettier doesnt work properly

File: Component/List/list.css (hunk 1 above)
Comment (sanjay-xdr) [reply Component/List/list.css:22]: i used color variable every where but this was a special case and used only here so didnt used color variables

File: Component/List/list.css [hunk 4]
```diff
@@ -0,0 +1,37 @@
+@import url("../../Utility CSS/navbar.css");
+
+.code-container {
+  width: 100%;
+}
+.box {
```
Comment (mohitdhatrak) [inline Component/List/list.css:6]: add a 1 line break/space here

File: Component/List/list.html [hunk 5]
```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (mohitdhatrak) [inline Component/List/list.html:6]: suggestion - add meta tags with author name and page description

File: Component/List/list.html [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+                <span class="nav__subtitle--position">Fast & Sharp UI</span>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <div class="container">
+      <div class="sidebar container-flex">
```
Comment (mohitdhatrak) [inline Component/List/list.html:37]: can use semantic elements like main and nav

File: Component/List/list.html [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+        <div class="margin-m2 padding-p2">
+          <code class="code-container container-flex flex-center">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28231%2C229%2C229%2C1%29&t=one-light&wt=none&l=text%2Fx-csrc&width=471.5&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=143%25&si=false&es=2x&wm=false&code=%2520%253Cul%253E%250A%2520%2520%253Cli%2520class%253D%2522list-item%2522%253EList%2520Item%25201%253C%252Fli%253E%2520%253Cbr%253E%250A%2520%2520%253Cli%2520class%253D%2522list-item%2522%253EList%2520Item%25202%253C%252Fli%253E%2520%253Cbr%253E%250A%2520%2520%253Cli%2520class%253D%2522list-item%2522%253EList%2520Item%25203%253C%252Fli%253E%2520%253Cbr%253E%250A%2520%253C%252Ful%253E"
+              style="
+                width: 100%;
+                height: 158px;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
```
Comment (mohitdhatrak) [inline Component/List/list.html:145]: if possible make a class and shift inline CSS to stylesheet

File: Component/Text Utilities/textUtilities.html [hunk 8]
```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (mohitdhatrak) [inline Component/Text Utilities/textUtilities.html:6]: same suggestion for meta tags

File: Component/Text Utilities/textUtilities.html [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+                <span class="nav__subtitle--position">Fast & Sharp UI</span>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <div class="container">
+      <div class="sidebar container-flex">
```
Comment (mohitdhatrak) [inline Component/Text Utilities/textUtilities.html:36]: same as above, use semantic tags e.g. nav or aside

**Review by mohitdhatrak — COMMENTED** [review]
Looks good, small suggestions here and there. Good work

**Pull request merged**